doc/user/formats.rst

```diff
@@ -130,3 +130,71 @@ fractional_ranges
         be masked if :math:`[\frac{i-0.5}{nchans}, \frac{i+0.5}{nchans}]`
         overlaps any of the ranges. Note that this means that channel 0
         is centred at 0.0 but extends below it.
+
+Primary beam
+------------
+The file contains the information in a compact Fourier-transform ("aperture
+plane") form. It comprises a 3-dimensional array J of Jones matrices. Another
+three arrays :math:`\nu`, :math:`y` and :math:`x` indicate the frequency (in
+Hz) and spatial position (in metres) of each sample along the axes.
+
+To determine the response at a sampled frequency :math:`\nu_f` and some
+direction, turn the direction into direction cosines :math:`l` and :math:`m`
+relative to the pointing centre (as defined in
+:class:`katsdpmodels.primary_beam.AltAzFrame`). Then the Jones
+matrix for the response is
+
+.. math:: \frac{1}{|x|\cdot|y|}
+          \sum_{j,k} e^{-2\pi i (x_j l + y_k m)\nu_f/c} J_{f,k,j},
+
+where :math:`|x|`, :math:`|y|` are the number of elements in the respective
+arrays and :math:`c` is the speed of light.
+The Jones matrices correspond to :attr:`.OutputType.JONES_HV`. Note the
+reversed axis order in accessing :math:`J`.
+
+To sample at an intermediate frequency, use linear interpolation along the
+frequency axis in the aperture plane.
+
+See :class:`katsdpmodels.primary_beam.PrimaryBeam` for other definitions and
+sign conventions.
+
+Attributes
+^^^^^^^^^^
+model_type
+    ``primary_beam``
+
+model_format
+    ``aperture_plane``
+
+antenna
+    The name of the antenna to which this model applies. Absent if this model
+    is not specific to a single antenna (the more generic ``model_target`` may
+    provide human-readable information about the range of applicable antennas).
+
+receiver
```

Comment:
What about `band` or is that simply an average receiver? Frequency is not enough to distinguish between e.g. L and S bands.

Reply:
I feel like it is less useful because it should always just match the band you requested (it doesn't seem likely that we'd produce models that are an average across different bands) so it doesn't tell you anything you didn't know when you asked for the model. `receiver` tells you new information if you requested the model just by antenna, and `antenna` will at least tell you whether you got an antenna-specific model or an average one.

But it's easy to add and can't really hurt, so I'll do it.

Reply:
Done.

Reply:
I see your point... I forgot that the band is used in the alias. But at least it makes the HDF5 file self-contained, for what it's worth :-)